kayla-himmel/solitaire
# Shuffle functionality

Ticket: https://trello.com/c/s7il1kAC/36-epic-algorithms-implement-deck-shuffling

To test:
1. Run project (I used 'npm run start').
2. Scroll past initial React image to the large image of a card on the bottom.
3. Click card, observe console log to see array of Cards being shuffled and returned.
